core/controllers/skill_editor.py

```diff
@@ -96,8 +96,6 @@ def get(self, skill_id):
             'interaction_templates': jinja2.utils.Markup(
                 interaction_templates),
             'dependencies_html': jinja2.utils.Markup(dependencies_html),
-            'ALLOWED_INTERACTION_CATEGORIES': (
-                feconf.ALLOWED_QUESTION_INTERACTION_CATEGORIES)
```

Comment:
For topics and skills, it is not `ALLOWED_INTERACTION_CATEGORIES` that is passed in but `ALLOWED_QUESTION_INTERACTION_CATEGORIES`, they are different. 
So, can you take that into account as well? As I don't think it has been yet.

Reply:
Sorry I didn’t notice the difference. Already fixed the issue. PTAL.